deploy/registration-service/registration-service.yaml

```diff
@@ -18,39 +18,38 @@ objects:
       name: registration-service
       namespace: ${NAMESPACE}
     rules:
-      - apiGroups:
```

Comment:
I guess it didn't rebase correctly. Please do not change this file.

Reply:
hmm, that's weird... will try to regenerate again

Reply:
You need to pull the latest version of registration-service and then regenerate

Reply:
Done, I've also made the other config changes as mentioned in https://github.com/codeready-toolchain/host-operator/pull/536